_dashboards/visualize/area.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: Using Area charts
+parent: Visualization types
+grand_parent: Visualize
+nav_order: 5
+---
+
+# Using Area charts
+
+An area chart is a line chart with the area between the line and the axis shaded with a color, and is a primary visualization type used to display time series data. You can create area charts in Dashboards using the Area visualization type or using the TSVB, Vega, or VisBuilder visualization tools. For this tutorial, you'll use the Area visualization type.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/aggregation-based-area.gif" alt="Demonstration of the area chart tutorial steps" height="100">
+
+# Try it: Create a simple aggregation-based area chart
+
+In this tutorial you'll create a simple area chart in the [OpenSearch Dashboards playground](https://playground.opensearch.org/app/home#/) using sample data and aggregations.
```

Comment:
I would not direct them to the playground. If you just add a couple of steps to add the flight data to dashboards, they can do it locally and save it. Then we don't need the note about the playground not saving the visualization.